stream-chat-android-ui-components/src/main/res/values/colors.xml

```diff
@@ -28,6 +28,7 @@
     <color name="stream_alabaster">#FCFCFC</color>
     <color name="stream_concrete">#F2F2F2</color>
     <color name="stream_grey_90">#E6E6E6</color>
+    <color name="stream_border_stroke">#14000000</color>
```

Comment:
Color meter shows #E6E6E6 which is defined above. Do you think we should use black with alpha or strive to avoid alpha?